zanata-war/src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -78,4 +78,387 @@
+      <if>#{identity.loggedIn}</if>
+      <to-view-id>/dashboard/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+  </navigation-rule>
+
+  <!-- Landing page for Kerberos ticket-based Authentication -->
+  <navigation-rule>
+    <from-view-id>/account/klogin.xhtml</from-view-id>
+    <navigation-case>
+      <if>#{authenticationManager.authenticationRedirect eq 'login'}</if>
+      <to-view-id>/account/login.xhtml</to-view-id>
+      <redirect />
+    </navigation-case>
+    <navigation-case>
+      <if>#{authenticationManager.authenticationRedirect eq 'edit'}</if>
+      <to-view-id>/profile/create_user.xhtml</to-view-id>
+      <redirect />
+    </navigation-case>
+    <navigation-case>
+      <if>#{authenticationManager.authenticationRedirect eq 'inactive'}</if>
+      <to-view-id>/account/inactive_account.xhtml</to-view-id>
+      <redirect />
+    </navigation-case>
+    <navigation-case>
+      <if>#{authenticationManager.authenticationRedirect eq 'dashboard'}</if>
+      <to-view-id>/dashboard/home.xhtml</to-view-id>
+      <redirect />
+    </navigation-case>
+    <navigation-case>
+      <if>#{authenticationManager.authenticationRedirect eq 'home'}</if>
+      <to-view-id>/home.xhtml</to-view-id>
+      <redirect />
+    </navigation-case>
+  </navigation-rule>
+
+  <!-- Landing page for Internal, JAAS and Kerberos form-based
+       authentication -->
+  <navigation-rule>
+    <from-view-id>/account/login.xhtml</from-view-id>
+    <navigation-case>
+      <from-action>#{loginAction.getLoginPageRedirect()}</from-action>
+      <from-outcome>dashboard</from-outcome>
+      <to-view-id>/dashboard/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <from-outcome>dashboard</from-outcome>
+      <to-view-id>/dashboard/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <from-outcome>createUser</from-outcome>
+      <to-view-id>/profile/create_user.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <from-outcome>inactive</from-outcome>
+      <to-view-id>/account/inactive_account.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <from-outcome>continue</from-outcome>
+      <to-view-id>#{userRedirect.url}</to-view-id>
+      <redirect/>
+    </navigation-case>
+  </navigation-rule>
+
+  <!--<navigation-rule>
+    <from-view-id>/account/password_reset_request.xhtml</from-view-id>
+    <navigation-case>
+      <from-outcome>home</from-outcome>
+      <to-view-id>/home.xhtml</to-view-id>
+      &lt;!&ndash; TODO [CDI] end conversation (it will lose notification message) &ndash;&gt;
+      &lt;!&ndash;<end-conversation />&ndash;&gt;
+      <redirect/>
+    </navigation-case>
+  </navigation-rule>-->
+
+  <navigation-rule>
+    <from-view-id>/account/register.xhtml</from-view-id>
+    <navigation-case>
+      <from-outcome>true</from-outcome>
```

Comment:
I think we should change it to a string. The navigation case is too general and makes it hard to understand what happens. I think in general (i.e. whenever possible) we should use the `<from-action>` parameter to be able to clearly understand how it's navigating.